UefiPayloadPkg/NetworkDrivers/dasharo.ipxe

```diff
@@ -4,13 +4,17 @@
 :MENU
 menu
 item --gap -- ------------------------ Dasharo Network Boot Menu ------------------------
+item boot           Autoboot (DHCP)
 item dasharo        Dasharo Tools Suite
 item netbootxyz     OS installation (netboot.xyz official server)
-item boot           Autoboot (DHCP)
 item shell          iPXE Shell
-choose --default dasharo --timeout 3000 target && goto ${target}
+choose --default boot --timeout 3000 target && goto ${target}
 
 # main menu options
+:boot
```

Comment:
This block doesn't have to change the location. It is simply a label with instruction to invoke when given label is reached. Only the `item` order needs change

Reply:
I know, I just thought it would make more sense to have the labels and boot menu items in the same order. Can revert though

Reply:
Sometimes too much unnecessary changes may introduce unwanted bugs :) Anyway let's treat it as an informational discussion. Closing.

Reply:
Understood :)